src/main/java/org/team1540/robot2018/commands/elevator/MoveElevatorToPosition.java

```diff
@@ -0,0 +1,10 @@
+package org.team1540.robot2018.commands.elevator;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class MoveElevatorToPosition extends CommandGroup {
+
+  public MoveElevatorToPosition(double target) {
+    addSequential(new MoveElevator(target));
```

Comment:
...wait a one command CommandGroup? Why? If there's a reason add a TODO

Reply:
That used to (and will soon) have wrist safety logic; I'll add a TODO for now.